### ISSUE TITLE
No data found. Please verify arguments

### ISSUE THREAD

**Issue by healthyfat** [kohya-ss/sd-scripts#78]
accelerate launch --num_cpu_threads_per_process=16 "train_network.py" --pretrained_model_name_or_path="C:/Programs/stable-diffusion-webui/models/Stable-diffusion/sd-v1-4.ckpt" --train_data_dir="C:/Users/pdept/Desktop/AI pics/training/512x512" --resolution=512,512 --output_dir="C:/Users/pdept/Desktop/AI pics/training/Nowy folder" --use_8bit_adam --xformers --logging_dir="" --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=1e-3 --network_dim=8 --output_name="last" --learning_rate="1e-5" --lr_scheduler="cosine" --train_batch_size="1" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --seed="1234" --cache_latents --max_data_loader_n_workers="1" --gradient_checkpointing --xformers --use_8bit_adam
prepare tokenizer
Use DreamBooth method.
prepare train images.
0 train images with repeating.
loading image sizes.
0it [00:00, ?it/s]
prepare dataset
No data found. Please verify arguments / 画像がありません。引数指定を確認してください

**Comment by healthyfat** [comment]
Thank you for suggestion. Unfortunately still same issue:

max_train_steps = 0
stop_text_encoder_training = 0
lr_warmup_steps = 0
accelerate launch --num_cpu_threads_per_process=16 "train_network.py" --pretrained_model_name_or_path="C:/Programs/stable-diffusion-webui/models/Stable-diffusion/sd-v1-4.ckpt" --train_data_dir="C:/Users/pdept/Desktop/AI pics/training/train data/img/100_wny style" --resolution=512,512 --output_dir="C:/Users/pdept/Desktop/AI pics/training/train data/model" --use_8bit_adam --xformers --logging_dir="" --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=1e-3 --network_dim=8 --output_name="wny" --learning_rate="1e-5" --lr_scheduler="cosine" --train_batch_size="1" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --seed="1234" --cache_latents --max_data_loader_n_workers="1" --xformers --use_8bit_adam
prepare tokenizer
Use DreamBooth method.
prepare train images.
0 train images with repeating.
loading image sizes.
0it [00:00, ?it/s]
prepare dataset
No data found. Please verify arguments / 画像がありません。引数指定を確認してください

**Comment by healthyfat** [comment]
Interesting. It works, when I point only to the folder where the folder with images is located. 

So instead of:

/img/100_wny style

I choose:

/img

Folder 100_wny style: 700 steps
max_train_steps = 700
stop_text_encoder_training = 0
lr_warmup_steps = 70
accelerate launch --num_cpu_threads_per_process=16 "train_network.py" --pretrained_model_name_or_path="C:/Programs/stable-diffusion-webui/models/Stable-diffusion/sd-v1-4.ckpt" --train_data_dir="C:/Users/pdept/Desktop/AI pics/training/train data/img" --resolution=512,512 --output_dir="C:/Users/pdept/Desktop/AI pics/training/train data/model" --use_8bit_adam --xformers --logging_dir="" --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=1e-3 --network_dim=8 --output_name="wny" --learning_rate="1e-5" --lr_scheduler="cosine" --lr_warmup_steps="70" --train_batch_size="1" --max_train_steps="700" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --seed="1234" --cache_latents --max_data_loader_n_workers="1" --xformers --use_8bit_adam
prepare tokenizer
Use DreamBooth method.
prepare train images.
found directory 100_wny style contains 7 image files
700 train images with repeating.
loading image sizes.
100%|███████████████████████████████████████████████████████████████████████████████████| 7/7 [00:00<00:00, 202.55it/s]
prepare dataset
prepare accelerator
Using accelerator 0.15.0 or above.
load StableDiffusion checkpoint
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>

**Comment by xbrontxx** [comment]
thanks for reporting the solution, I had the same problem, works now

**Comment by angelOnly** [comment]
The first directory that must be created under the train directory is the location of the image data, and within this directory you need to create another directory named with the number _ name, for example here I wrote sweet, 2_sweet, train_data_dir=./train/sweet, Instead of train_data_dir=./train/sweet/2_sweet, the command here is very important. The number represents how many times the material in the folder is used

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/18018532/232045051-931f45ac-94a8-4df4-9164-5f0b08c2e58b.png">
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/18018532/232046071-002c7da6-973e-45f6-aefe-f8faac89680e.png">
